File: Patterns in C/Pattern 9.cpp
# include <stdio.h>
int main()
{
	int i,j,n;
	printf("Enter No of Rows ");
	scanf("%d",&n);
	
	for(i=0;i<n;i++)
	{
		for(j=1;j<=n;j++)
		{
			printf("%d ",j);
		}
		printf("\n");
	}
	return 0;
}

File: Patterns in Java/Pattern_9.java
import java.util.Scanner;

public class Pattern_9 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter No of Rows: ");
    int n = sc.nextInt();
    
    for (int i = 0; i < n; i++) {
      for (int j = 1; j <= n; j++) {
        System.out.print(j + " ");
      }
      System.out.println();
    }
  }
}